bundle/manifests/dell-csm-operator.clusterserviceversion.yaml

```diff
@@ -738,7 +738,7 @@ metadata:
                         "value": "powermax"
                       }
                     ],
-                    "image": "dellemc/dell-csi-replicator:v1.6.0",
+                    "image": "dellemc/dell-csi-replicator:v1.7.0",
```

Comment:
Could you make similar changes for powerscale and powerflex in this file?

Reply:
Done